Tests/Shopware/Connect/Component/AutoCategoryFixerTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Tests\ShopwarePlugins\Connect\Component;
+
+
+use ShopwarePlugins\Connect\Components\AutoCategoryFixer;
+
+class AutoCategoryFixerTest extends \PHPUnit_Framework_TestCase
+{
+    public function testRecreateRemoteCategories()
+    {
+        $importService = $this->getMockBuilder('\\ShopwarePlugins\\Connect\\Components\\ImportService')
```

Comment:
You used the string again instead of `\ShopwarePlugins\Connect\Components\ImportService::class` :D
And normally you could use `$this->createMock(ImportService::class)` directly. It is a little wrapper method provide by `PhpUnit` which disables the `constructor` directly and creates the mock ;)